internal/controllers/clusterrequest_controller.go

```diff
@@ -612,59 +646,77 @@ func (t *clusterRequestReconcilerTask) handleClusterInstallation(ctx context.Con
 		existingClusterInstance)
 	if err != nil {
 		if !errors.IsNotFound(err) {
-			return fmt.Errorf("failed to get ClusterInstance: %w", err)
+			return nil, fmt.Errorf("failed to get ClusterInstance: %w", err)
+		}
+
+		operationType := "created"
```

Comment:
I think the `created` and `validated with dry-run` values for `operationType` can be constants.